chia/util/initial-config.yaml

```diff
@@ -483,6 +483,11 @@ wallet:
   # see description for full_node.db_sync
   db_sync: auto
 
+  # the number of threads used to read from the wallet database
+  # concurrently. There's always only 1 writer, but the number of readers is
+  # configurable
+  db_readers: 2
```

Comment:
I don't think the wallet has as much pressure from tasks wanting to read from its DB, so two reader threads seems sufficient.